00_source_data/data.py

```diff
@@ -0,0 +1,155 @@
+"""
+importing all raw data for the analysis and filtering the useful columns
+"""
+import pandas as pd
+import datetime as dt
+
+# Fips codes csv
+fips = pd.read_csv(
+    "https://raw.githubusercontent.com/kjhealy/fips-codes/master/state_and_county_fips_master.csv"
+)
+fips.sample(6)
+fips.columns
+
+# Opioids Description Dataset
+# url = "https://d2ty8gaf6rmowa.cloudfront.net/dea-pain-pill-database/bulk/arcos_all_washpost.tsv.gz"
+url = r"C:\Users\ericr\Downloads\cmder\720newsafeopioids\arcos_all_washpost.tsv"
+
```

Comment:
Thanks for the codes! Probably it's better to sort out and generate a dataframe for opioids from local end and upload a smaller size of data frame onto git hub so that we can all access the data frame and run the following steps (merging, analysis)?